monitor-services/bgpstreamkafkatap/core/bgpstreamkafka.py

```diff
@@ -364,6 +362,42 @@ def start_rest_app(self):
         IOLoop.current().start()
 
 
+class RedisChecker:
```

Comment:
this would make sense to be in the `artemis-utils` package right?

Reply:
You are right I will move it there

Reply:
But how do I pass the stop_data_worker function which is custom? I should add it to the args probably and simply use it.

Reply:
@slowr done it works OK, please re-check PR when available and let me know. The travis tests are not failed, they have reached pull rate limits (will re-issue them later)